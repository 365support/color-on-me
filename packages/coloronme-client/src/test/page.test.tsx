import { render, screen } from '@testing-library/react';
import Page from '@/src/pages/index';

describe('predict page', () => {
  it('이건 프레딕트 글자가 잘 나오는지 체크', () => {
    const { container } = render(<Page />);

    expect(container).toMatchSnapshot();
  });
});
