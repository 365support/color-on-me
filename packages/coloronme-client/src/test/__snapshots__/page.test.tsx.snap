// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`predict page 이건 프레딕트 글자가 잘 나오는지 체크 1`] = `
<div>
  <h1>
    hello
  </h1>
</div>
`;
